src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Experimental.cs

```diff
@@ -0,0 +1,602 @@
+                //         F(default(SC));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "SC").WithArguments("SC", "OC").WithLocation(8, 19),
+                // (9,19): warning CS0618: 'SD' is obsolete: 'DC'
+                //         F(default(SD));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "SD").WithArguments("SD", "DC").WithLocation(9, 19),
+                // (10,19): error CS0619: 'SE' is obsolete: 'DD'
+                //         F(default(SE));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "SE").WithArguments("SE", "DD").WithLocation(10, 19),
+                // (11,19): error CS0619: 'SF' is obsolete: 'OF'
+                //         F(default(SF));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "SF").WithArguments("SF", "OF").WithLocation(11, 19),
+                // (12,19): warning CS0618: 'IG' is obsolete: 'DG'
+                //         F(default(IG));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "IG").WithArguments("IG", "DG").WithLocation(12, 19),
+                // (13,19): warning CS0618: 'IH' is obsolete: 'DH'
+                //         F(default(IH));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "IH").WithArguments("IH", "DH").WithLocation(13, 19),
+                // (14,19): warning CS0618: 'II' is obsolete: 'DI'
+                //         F(default(II));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "II").WithArguments("II", "DI").WithLocation(14, 19),
+                // (15,19): error CS0619: 'IJ' is obsolete: 'DJ'
+                //         F(default(IJ));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "IJ").WithArguments("IJ", "DJ").WithLocation(15, 19),
+                // (16,19): error CS0619: 'IK' is obsolete: 'DK'
+                //         F(default(IK));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "IK").WithArguments("IK", "DK").WithLocation(16, 19),
+                // (17,19): warning CS0618: 'EM' is obsolete: 'OM'
+                //         F(default(EM));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "EM").WithArguments("EM", "OM").WithLocation(17, 19),
+                // (18,19): error CS0619: 'EN' is obsolete: 'ON'
+                //         F(default(EN));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "EN").WithArguments("EN", "ON").WithLocation(18, 19),
+                // (19,19): warning CS0618: 'EO' is obsolete: 'DO'
+                //         F(default(EO));
+                Diagnostic(ErrorCode.WRN_DeprecatedSymbolStr, "EO").WithArguments("EO", "DO").WithLocation(19, 19),
+                // (20,19): error CS0619: 'EP' is obsolete: 'DP'
+                //         F(default(EP));
+                Diagnostic(ErrorCode.ERR_DeprecatedSymbolStr, "EP").WithArguments("EP", "DP").WithLocation(20, 19));
+        }
+
+        // Suppress diagnostics in unused import statements.
```

Comment:
>// Suppress diagnostics in unused import statements. [](start = 8, length = 52)

Is this comment accurate? It looks like the obsolete diagnostics is always suppressed within alias targets, regardless whether aliases are used or not.